docker-compose.test.yml

```diff
@@ -25,7 +25,7 @@ services:
             - DATABASE_MODE=test
             - FEATURE_FLAG=oss
             - OPENAI_API_KEY=${OPENAI_API_KEY}
-            - AGENTA_TEMPLATE_REPO=agentaai/templates_v2
+            - AGENTA_TEMPLATE_REPO=agentaai/stage_templates
```

Comment:
why is this part of the pr

Reply:
Same as [this](https://github.com/Agenta-AI/agenta/pull/1705#discussion_r1624181360).

Reply:
shouldn't this be reverted to use the last templates (otherwise PR tests would fail if someone is testing a staging template)

Reply:
The `stage_templates` will always be in sync with `templates_v2`, correct? 

In most cases, like this one currently, we updated the templates based on the changes we have made to the SDK. With that said, testing the Cypress tests while it's using the default compose will not work, since the default compose only makes use of `templates_v2` and not `stage_templates` that has the SDK changes.

To answer your concern, I'd say no. We should leave things as they are. 

> otherwise PR tests would fail if someone is testing a staging template

I don't understand this. The staging template is only used during tests with the test compose file, not in the default compose (that users are using). And as I have mentioned earlier, the staging template will always be in sync with the production templates, and in most cases, changes we have made to the SDK would need testing, and the only way to test that the Cypress tests are passing is to make use of the test compose with `stage_templates` repository.

Reply:
The `stage_templates` will always be in sync with `templates_v2`, correct? 

In most cases, like this one currently, we updated the templates based on the changes we have made to the SDK. With that said, testing the Cypress tests while it's using the default compose will not work, since the default compose only makes use of `templates_v2` and not `stage_templates` that has the SDK changes.

To answer your concern, I'd say no. We should leave things as they are. 

> otherwise PR tests would fail if someone is testing a staging template

I don't understand this. The staging template is only used during tests with the test compose file, not in the default compose (that users are using). And as I have mentioned earlier, the staging template will always be in sync with the production templates, and in most cases, changes we have made to the SDK would need testing, and the only way to test that the Cypress tests are passing is to make use of the test compose with `stage_templates` repository.

Reply:
Good points. I agree.